src/coordinate/ModuleGraph.zig

```diff
@@ -7,7 +7,8 @@ const base = @import("../base.zig");
 const utils = @import("utils.zig");
 const cache = @import("../cache.zig");
 const collections = @import("../collections.zig");
-const can = @import("../check/canonicalize.zig");
+const Can = @import("../check/canonicalize.zig");
```

Comment:
TitleCase implies type names in Zig, but this is just a module.

Reply:
not anymore 😉